torch/csrc/lazy/core/lazy_graph_executor.cpp

```diff
@@ -936,6 +930,16 @@ std::shared_ptr<LazyGraphExecutor::Async> LazyGraphExecutor::
         std::vector<BackendDataPtr> parameters_data,
         std::vector<BackendDataPtr> tensors_data,
         ComputationCache::TypePtr cached_computation) {
+  if (coll) {
+    VLOG(4) << "Waiting on device barrier for device " << coll->device << " ...";
```

Comment:
@JackCaoG  Since we are here, let me ask this question. I found the way to turn on VLOG is pretty awkward . Do you guys find this kind of VLOG msg helpful? There are not many of them, so if nobody cares, we can get rid of them in LTC.

Reply:
it is pretty useful when debugging "My program is stuck" kind of issue. It give people a good sense of what is actually happening.